### PR TITLE
Updated data types and hello world lessons

### PR DESCRIPTION
**Pull request by v1g0rous**

### CODE REVIEW

File: JavaCore/lesson_data_types_operators.MD [hunk 1]
```diff
@@ -38,6 +94,10 @@ String name = "John";
 String[] names = {"John", "Doe", "Jane"};
 ```
 
+### 1.3 Difference between primitives and reference data types
+- TO DO: add what's the difference between primitives and reference with example (e.g. reference type can be NULL)
+- TO DO: explain what does it mean "pass by reference/pass by value" and give examples with both types (primitive and reference)
```
Comment (temirlan100) [inline JavaCore/lesson_data_types_operators.MD]: Good point! But need to add info about it or remove

No need TO DO in the master branch

Comment (v1g0rous) [reply JavaCore/lesson_data_types_operators.MD]: OK, what if I have to do points, but don't have text yet, then how to share them with you?
In TG or somehow else?

Comment (temirlan100) [reply JavaCore/lesson_data_types_operators.MD]: Great! You can remove it from here and add an issue than create a new PR

Comment (v1g0rous) [reply JavaCore/lesson_data_types_operators.MD]: Done

**Pull request merged**